deepchem/data/datasets.py

```diff
@@ -2109,16 +2274,16 @@ def select(self, indices: Sequence[int],
 
     Parameters
     ----------
-    indices: list
+    indices: Sequence
       List of indices to select.
-    select_dir: Optional[str], (default None)
+    select_dir: str, optional (default None)
       Path to new directory that the selected indices will be copied
       to.
 
     Returns
     -------
     DiskDataset
-      Contains selected indices.
+      A selected DiskDataset object
```

Comment:
How about, "A Dataset containing the selected samples"?

Reply:
I fixed using it!